apps/docs/content/components/navigation/Disclosure.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Disclosure
```

Comment:
![image](https://github.com/user-attachments/assets/5094f06e-ffff-431f-93ae-fa4998edc3a0)
We discussed it but not sure why isDisabled is inside Accessibility props

Reply:
yep, this will be fixed by one of us in another pr.